src/components/cylc/gantt/GanttChart.vue

```diff
@@ -0,0 +1,218 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <VueApexCharts
+    type="rangeBar"
+    :options="chartOptions"
+    :series="series"
+    width="100%"
+    height="auto"
+    class="d-flex justify-center"
+  />
+  <v-pagination
+    v-model="page"
+    :length="numPages"
+    :total-visible="7"
+    density="comfortable"
+  />
+</template>
+
+<script>
+import VueApexCharts from 'vue3-apexcharts'
+import {
+  mdiDownload,
+} from '@mdi/js'
+import { useReducedAnimation } from '@/composables/localStorage'
+
+export default {
+  name: 'GanttChart',
+
+  watch: {
+    tasksPerPage: function () {
+      this.page = 1
+    },
+  },
+  components: {
+    VueApexCharts,
+  },
+
+  props: {
+    jobs: {
+      type: Array,
+      required: true,
+    },
```

Comment:
Just spotted this type is incorrect
```suggestion
    /** @type {Record<string, Object[]>} */
    jobs: {
      type: Object,
      required: true,
    },
```